### PR TITLE
Currency : added ubi9.3 script for jgrapht

### PR DESCRIPTION
**Pull request by stutiibm**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: j/jgrapht/jgrapht_ubi_9_3.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+export PACKAGE_NAME=jgrapht
+export PACKAGE_URL=https://github.com/jgrapht/jgrapht
+export LC_ALL=C.UTF-8
```
Comment (nikhil-kalbande) [inline j/jgrapht/jgrapht_ubi_9_3.sh:25]: Can you add comment to provide reason for adding these extra steps so for non travis infra if these are not needed user can comment these lines

File: j/jgrapht/build_info.json [hunk 2]
```diff
@@ -2,26 +2,17 @@
...
-    },
-
-    "1.4.0": {
-       "build_script": "jgrapht_ubi_8_7.sh"
-    },
-
-    "jgrapht-1.3.*": {
-       "build_script": "jgrapht_ubi_8_7.sh"
+     "jgrapht-1.5.*": {
+       "build_script": "jgrapht_ubi_9_3.sh"
```
Comment (nikhil-kalbande) [inline j/jgrapht/build_info.json:13]: @stutiibm so will current script works for older version also. We need to keep support for older version also be it through ubi 8 script

Comment (stutiibm) [reply j/jgrapht/build_info.json:19]: Added comment in script

Comment (stutiibm) [reply j/jgrapht/build_info.json:19]: Added support for older versions

File: j/jgrapht/jgrapht_ubi_9_3.sh (hunk 1 above)
Comment (stutiibm) [reply j/jgrapht/jgrapht_ubi_9_3.sh:27]: Added comment in script

**Review by nikhil-kalbande — APPROVED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**